.rubocop.yml

```diff
@@ -26,3 +26,7 @@ Style/TrailingCommaInLiteral:
 
 Style/TrivialAccessors:
   AllowPredicates: true
+
+Metrics/BlockLength:
+  Exclude:
+    - 'spec/**/*.rb'
```

Comment:
See https://stackoverflow.com/questions/40934345/rubocop-25-line-block-size-and-rspec-tests for background.

Reply:
Thanks! This number could be reduced then: https://github.com/kirs/mysql2/blob/b50e1178fe1f8db5fc252a3697b94b42a0f34db2/.rubocop_todo.yml#L24